shared/fs/folder/index.js

```diff
@@ -30,6 +33,7 @@ const WithContent = (props: Props) => (
         routePath={props.routePath}
         headerRows={[
           ...resetBannerAsRows(props.path, props.resetBannerType),
+          ...conflictBannerAsRows(props.path, props.conflictState),
```

Comment:
I'm inclined to having this as a fixed banner rather than a row that scrolls with the content. WDYT?

Reply:
Yeah I think you're right.. we should have a more standard system for dealing with all our banners.. I feel like I've been just sticking them semirandomly into different levels of the component hierarchy depending on where they appear in the UI, but this seems suboptimal.